src/Parser/Shaman/Elemental/Modules/Talents/ExposedElements.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import { formatNumber, formatPercentage } from 'common/format';
+import SpellIcon from 'common/SpellIcon';
+import SPELLS from 'common/SPELLS';
+import calculateEffectiveDamage from 'Parser/Core/calculateEffectiveDamage';
+
+
+class ExposedElements extends Analyzer {
+  static dependencies = {
+    combatants : Combatants,
+    enemies : Enemies,
+  };
+
+  EXPOSED_ELEMENTS  = {
+    INCREASE : 1,
+    WINDOW_DURATION: 300,
+  };
+
+  removeDebuffTimestamp = null;
+  damageGained=0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.EXPOSED_ELEMENTS_TALENT.id);
+  }
+
+  on_byPlayer_cast(event) {
+
+    if(event.ability.guid !== SPELLS.LIGHTNING_BOLT.id){
+      return;
+    }
+
+    const enemy = this.enemies.getEntity(event);
+    console.log(enemy);
+    if((enemy) && (enemy.hasBuff(SPELLS.EXPOSED_ELEMENTS_DEBUFF.id))){
```

Comment:
you could remove some of these parentheses:
`if(enemy && enemy.hasBuff(SPELLS.EXPOSED_ELEMENTS_DEBUFF.id)) {`